error-handler.js

```diff
@@ -269,6 +277,9 @@ createHandler = function createHandler(options) {
       return resumeOrClose(status);
     }
 
+    // Always set a response status.
```

Comment:
Yes that was my line of thought, but I wanted to initialize the [status response](https://github.com/strongloop/express/blob/63ab25579bda70b4927a179b580a9c580b6c7ada/lib/response.js#L41), as seen in the [PR](https://github.com/localnerve/express-error-handler/blob/b90babea3ce84c8eb97fd56aa8404b67d62dc208/error-handler.js#L281), to cover all the response cases that need it at that point (custom-handler, view, static).
The response status code [isn't changing again](https://github.com/localnerve/express-error-handler/blob/b90babea3ce84c8eb97fd56aa8404b67d62dc208/error-handler.js#L112), except in the [fall-through case](https://github.com/localnerve/express-error-handler/blob/b90babea3ce84c8eb97fd56aa8404b67d62dc208/error-handler.js#L318).

My idea was to set this [sooner](https://github.com/localnerve/express-error-handler/blob/b90babea3ce84c8eb97fd56aa8404b67d62dc208/error-handler.js#L281), and then conform it only in the [fall-through case](https://github.com/localnerve/express-error-handler/blob/b90babea3ce84c8eb97fd56aa8404b67d62dc208/error-handler.js#L318). 
The status is already set from error or response status [at the beginning of errorHandler](https://github.com/localnerve/express-error-handler/blob/b90babea3ce84c8eb97fd56aa8404b67d62dc208/error-handler.js#L211). It's just not carried to the response object yet. The downstream setting of this doesn't add anything, it [isn't changing again](https://github.com/localnerve/express-error-handler/blob/b90babea3ce84c8eb97fd56aa8404b67d62dc208/error-handler.js#L112),  just delays setting it and unfortunately bypasses the non-`renderDefault` cases (customHandler, view, and static).

Also please consider that the 500 conformation in the fall-through can hide crucial status information (511 for example). I would conform the response status to 500 only if the response is not already in the 500 range, thus preserving any error nuance that might be helpful.